src/components/collectibles/CollectibleGallery.jsx

```diff
@@ -0,0 +1,74 @@
+import { h } from 'preact'
+import { useState, useEffect } from 'preact/hooks'
+import cn from 'classnames'
+import Card from '../card/Card'
+import styles from './CollectibleGallery.module.css'
+import CollectibleDetailsView from './CollectibleDetailsView'
+import CollectiblesHeader from './CollectiblesHeader'
+import CollectibleTile from './CollectibleTile'
+import { fetchJsonFromCID } from '../../util/fetchCID'
+
+const CollectibleGallery = ({
+  collectibles,
+  user,
+  isTwitter,
+  backgroundColor
+}) => {
+  const [modalCollectible, setModalCollectible] = useState({})
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  const [order, setOrder] = useState([])
+
+  const fetchCollectiblesOrder = async () => {
```

Comment:
since we're using this in an effect, this should be a useCallback w/ dependencies [user, setOrder] and that means the useEffect below should have a dependency on fetchCollectiblesOrder